PyPortal_Word_of_the_Day/code.py

```diff
@@ -20,7 +20,11 @@
 WORD_LOCATION = ['word']
```

Comment:
This doesn't need to be predefined, just use `['word']` on Line 62.

Reply:
kk!